src/org/pentaho/reporting/platform/plugin/SimpleReportingComponent.java

```diff
@@ -103,6 +104,7 @@
   public static final String DASHBOARD_MODE = "dashboard-mode"; //$NON-NLS-1$
   private static final String MIME_GENERIC_FALLBACK = "application/octet-stream"; //$NON-NLS-1$
   public static final String PNG_EXPORT_TYPE = "pageable/X-AWT-Graphics;image-type=png";
+  public static final String QUERY_LIMIT = "query-limit"; //$NON-NLS-1$
```

Comment:
Kill this duplicate string, use ParameterXmlContentHandler.SYS_PARAM_QUERY_LIMIT in its place. This way there is only one source of truth.